PeripheryFunctions/TS_GetIDs.m

```diff
@@ -1,16 +1,19 @@
-function [IDs,notIDs] = TS_GetIDs(theKeyword,whatData,tsOrOps)
+function IDs = TS_GetIDs(theFields,whatData,tsOrOps,nameOrKeywords)
```

Comment:
Why remove the `notIDs` output? This is needed in other functions (like `TS_FilterData`)

Reply:
Outputting a sorted list doesn't really mesh with also outputting `notIDs`. Could `TS_FilterData` use the NaN's to know which are notIDs outside of the function?

Reply:
Did you see my latest commits on this (temporary)? I think a set of IDs as originally formulated is clearest. What are the NaNs labeling? I think it's important to give results as IDs that can be matched to across different versions of a dataset (rather than say a binary index label that is specific to the index/size of a specific dataset).

Reply:
Just seen the latest commits. So it seems we can either go about it this way (with one TS_GetIDs) or have a TS_GetIDs (your original function) and another TS_GetIDsFromNames, which returns the sorted list. The NaNs are when we don't find a match for that particular name.

Reply:
I think it makes sense to keep these as two modes within a single function… (if it gets clunky we can always split into two functions)